### PR TITLE
Prisma set up

### PR DESCRIPTION
**Pull request by choobs1**
Set up a Prisma client and Prisma schema. Basic resolvers are setup as well

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB (Preview) and CockroachDB (Preview).
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+NODE_ENV="development"
+APP_SECRET="jwtsecret123"
+NODE_PATH="./src"
+DATABASE_URL="postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
```
Comment (bridges-wood) [inline .env]: .env files should NEVER be committed to a public repo. This is a security concern and I need you to add it to the `.gitignore`. Also you'll need to change the APP_SECRET and probably update the DB password.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,73 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = "postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
```
Comment (bridges-wood) [inline prisma/schema.prisma]: This URL contains the database password and should not be made public.

File: src/index.ts [hunk 3]
```diff
@@ -1,16 +1,67 @@
...
 
+const businesses : Pick<Business, 'name' | 'sustainabilityScore' | 'customerScore' | 'profilePicture' | 'profileText'>[] = [
+    {
+      name: "John",
+      sustainabilityScore: 1.078,
+      customerScore: 5.955,
+      profilePicture: "Sample.JPEG",
+      profileText: "John",
+    }
+  ]
```
Comment (bridges-wood) [inline src/index.ts]: This can be removed.

File: src/index.ts [hunk 4]
```diff
@@ -1,16 +1,67 @@
...
+  graphiql: true
+})
+
+/*
+app.get('/', async function (req, reply) {
+  // Get the query from the request body
+  const query = '{ add(x: 2, y: 2) }'
+  return reply.graphql(query)
+})
+*/
```
Comment (bridges-wood) [inline src/index.ts]: This can be removed.

File: src/index.ts [hunk 5]
```diff
@@ -1,16 +1,67 @@
...
 }
 
-const server = new ApolloServer({typeDefs, resolvers})
+app.register(mercurius, {
+  schema,
+  resolvers,
+  context: (request, reply) => {
+    return { prisma }
+  },
+  graphiql: true
```
Comment (bridges-wood) [inline src/index.ts]: This needs to change depending on `NODE_ENV` - we don't want it available in production.

File: tsconfig.json [hunk 6]
```diff
@@ -9,6 +9,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
+    "noImplicitAny": false,
```
Comment (bridges-wood) [inline tsconfig.json]: This should be true.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Security critical changes needed.

File: .env (hunk 1 above)
Comment (choobs1) [reply .env]: "NODE_ENV", "APP_SECRET"  and "NODE_PATH" were added while following an issue with database connectivity. They will be removed now.

File: src/index.ts (hunk 5 above)
Comment (choobs1) [reply src/index.ts]: What do you mean by this?

File: src/index.ts (hunk 4 above)
Comment (choobs1) [reply src/index.ts]: It has been done

File: prisma/schema.prisma (hunk 2 above)
Comment (choobs1) [reply prisma/schema.prisma]: URL added to .env file

File: tsconfig.json (hunk 6 above)
Comment (choobs1) [reply tsconfig.json]: Has been changed to true in new commit

**Review by bridges-wood — APPROVED** [review]

**Pull request merged**